content/08.outlook.md

```diff
@@ -1 +1,41 @@
+The low number of samples while significant is not the only roadblock towards application of ML in rare disease data.
+The high dimensionality of the data collected using modern techniques necessitates the management of the curse of dimensionality in creative ways by learning low dimensional representations from the data.
+It further requires leveraging prior knowledge and transfer learning instead of using traditional machiine learning methods for appropriate interpretation of data. 
+Lastly, anyone applying machine learning methods on rare disease data is advised to delve into fundamental techniques that increase confidence and penalize complexity of the resultant models to enhance generalizabilty of ML in rare disease.
+All of the approaches highlighted in this perspective come with certain challenges or inadequacies that breed mistrust in using these powerful techniques in rare disease.
+We believe that the same challenges that are currently outlined as major pitfalls in applying ML to rare disease can be great opportunities for data generation as well as method development in moving the field forward.
+During our journey through the various challenges, we identified two major areas where mindful strategies can immeasurably enhance the power of machine learning in rare disease and move the field forward.
+
+_Emphasis on not just "more n" but "more meaningful n"._
+
+Mindful addition of data is key for powering the next generation of analysis in rare disease data.
+While there are many techniques to collate rare data from different sources, incorrect data generation may hurt the end goal even if it adds to the size of the dataset.
+In our experience collaboration with domain experts have proved to be critical in gaining insight into potential sources of variation in the datasets.
+As an example, in a study of neurofibromatosis type 1 (NF1) some samples were generated using vastly different surgical techniques (laser ablation and excision vs standard excision). [@doi:10.3390/genes11020226] 
+This resulted in surfacing of substantial biological differences that are a consequence of process (e.g. activation of heat shock protein related pathways), not disease related biology. 
+Prior collaboration with domain experts would have helped minimize these differences and generate a more robust dataset.
+A few such collaborations are beginning to form and show promise in generating valuable datasets for future use.[@https://www.hopkinsmedicine.org/kimmel_cancer_center/cancers_we_treat/pediatric_oncology/research_and_clinical_trials/pratilas/nf1_biospecimen_repository.html]
+
+Rare disease not only lacks samples, there is also a great dearth of comprehensive phenotypic-genotypic databases. 
+With the ubiquity of sequencing platforms, genomic data has been easy to gather for rare disease patients.
+This is augmented by the fact that most rare diseases seem to be genetic in origin.
```

Comment:
Citation? May be covered by `[TODO: add Refs]` below 😄

Reply:
:)